README.md

```diff
@@ -399,64 +140,97 @@ render() {
+| [**breakpoints**](https://brainhubeu.github.io/react-carousel/docs/examples/responsive) | *Object* | All props (except value, onChange, responsive, children) can be set to different values on different screen resolution |
+| [**centered**](https://brainhubeu.github.io/react-carousel/docs/examples/centered) | *Boolean* | Alignes active slide to the center of the carousel |
+| [**clickToChange**](https://brainhubeu.github.io/react-carousel/docs/examples/clickToChange) | *Boolean* | Clicking on a slide changes current slide to the clicked one |
+| [**dots**](https://brainhubeu.github.io/react-carousel/docs/examples/simpleDots) | *Boolean* | Renders default dots under the carousel |
+| [**draggable**](https://brainhubeu.github.io/react-carousel/docs/examples/draggable) | *Boolean* | Makes it possible to drag to the next slide with mouse cursor |
+| [**infinite**](https://brainhubeu.github.io/react-carousel/docs/examples/infinite) | *Boolean* | Creates an infinite carousel width |
+| **itemWidth** | *Number* | Determines custom width for every slide in the carousel |
+| [**keepDirectionWhenDragging**](https://brainhubeu.github.io/react-carousel/docs/examples/draggable) | *Boolean* | While dragging, it doesn't matter which slide is the nearest one, but in what direction you dragged |
+| **minDraggableOffset** | *Number* | Defines the minimum offset to consider the drag gesture |
+| **offset** | *Number* | Padding between items |
+| [**onChange**](https://brainhubeu.github.io/react-carousel/docs/examples/controlled) | *Function* | Handler triggered when current slide is about to change (e.g. on arrow click or on swipe) |
+| [**rtl**](https://brainhubeu.github.io/react-carousel/docs/examples/rtl) | *Boolean* | Iindicating if the carousel should have direction from Right to Left (make sure to pass the rtl param to the Dots component as well) |
+| **slides** | *Array* | Alternative way to pass slides. This prop expects an array of JSX <img> elements |
+| [**slidesPerPage**](https://brainhubeu.github.io/react-carousel/docs/examples/multipleItems) | *Number* | Number of slides visible at once |
+| [**slidesPerScroll**](https://brainhubeu.github.io/react-carousel/docs/examples/slidesPerScroll) | *Number* | Number by which value will change on scroll (autoPlay, arrow click, drag)|
+| **stopAutoPlayOnHover** | *Boolean* | Determines if autoPlay should stop when mouse hover over carousel |
+| [**value**](https://brainhubeu.github.io/react-carousel/docs/examples/controlled) | *Number* | Current slide's index (zero based, depends on the elements order) |
+
+### Dots props
+
+| Prop | Type | Description |
+| --- | --- | --- |
+| [**number**](https://brainhubeu.github.io/react-carousel/docs/examples/dots) | *Number* | Number of slides in carousel you want to control. By default it will be set to the number of thumbnails (if provided) |
```

Comment:
```suggestion
| [**number**](https://brainhubeu.github.io/react-carousel/docs/examples/dots) | *Number* | Number of slides in the carousel you want to control. By default, it will be set to the number of thumbnails (if provided) |
```